index.js

```diff
@@ -4,15 +4,30 @@
 module.exports = {
   name: 'ember-hbs-minifier',
 
-  setupPreprocessorRegistry(type, registry) {
-    if (type === 'parent') {
-      let HbsMinifierPlugin = require('./hbs-minifier-plugin');
+  _getMinifierOptions() {
+    return (this.parent && this.parent.options) || (this.app && this.app.options) || {};
+  },
```

Comment:
I guess we should probably use something like https://github.com/simplabs/ember-test-selectors/blob/master/index.js#L107-L118 to find the correct options hash 🤔

Reply:
I'll make the necessary changes here.